### ISSUE TITLE
dylibs link for M1 (arm64)

### ISSUE THREAD

**Issue by Mikasa-san** [iina/iina#4405]
so when I set the link to https://iina.io/dylibs/arm64 it shows error 
<img width="570" alt="Screenshot 2023-05-10 at 1 20 48" src="https://github.com/iina/iina/assets/54586069/0cfc9a6d-a9c9-4455-8f38-9207dc2ebfbd">

**Comment by smishralm10** [comment]
Have you tried it with `universal`? My development machine has M2 chip and `universal` worked fine for me.

**Comment by Mikasa-san** [comment]
> Have you tried it with `universal`? My development machine has M2 chip and `universal` worked fine for me.

Yeah, universal is works but I want arm64 only and it was working about 2 months.
